pkg/controller/plan/adapter/openstack/client.go

```diff
@@ -955,7 +969,7 @@ func (r *Client) ensureImageUpToDate(vm *libclient.VM, image *libclient.Image, v
 			"vm", vm.Name, "image", inventoryImage.Name, "properties", inventoryImage.Properties)
 	}
 
-	return
+	return upToDate, nil
```

Comment:
I'd suggest to drop the named out arguments then, but again - let's minimize the changes here so please post it separately